grr/server/grr_response_server/gui/ui/components/flow_details/plugins/osquery_details.ng.html

```diff
@@ -10,6 +10,18 @@
 </div>
 
 <ng-container *ngIf="flowCompleted$ | async">
+  <div
+    *ngIf="atLeastOneRowAvailable$ | async"
+    class="flow-details-summary-actions export-button">
+    <a
```

Comment:
Please add the [`download`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) attribute.